src/gui/guiFormSpecMenu.cpp

```diff
@@ -1382,23 +1533,35 @@ void GUIFormSpecMenu::parseTabHeader(parserData* data, const std::string &elemen
 {
 	std::vector<std::string> parts = split(element,';');
 
-	if (((parts.size() == 4) || (parts.size() == 6)) ||
-		((parts.size() > 6) && (m_formspec_version > FORMSPEC_API_VERSION)))
+	if (((parts.size() == 4) || (parts.size() == 6)) || (parts.size() == 7 &&
+		data->real_coordinates) || ((parts.size() > 6) &&
+		(m_formspec_version > FORMSPEC_API_VERSION)))
 	{
 		std::vector<std::string> v_pos = split(parts[0],',');
-		std::string name = parts[1];
-		std::vector<std::string> buttons = split(parts[2],',');
-		std::string str_index = parts[3];
+
+		// If we're using real coordinates, add an extra field for height.
+		// Width is not here because tabs are the width of the text, and
+		// there's no reason to change that.
+		int i = 0;
+		std::string h_geom;
+		if (data->real_coordinates) {
+			i++;
+			h_geom = parts[1];
```

Comment:
What is if the 6 parameter element is used with `real_coordinates`?

Reply:
I don't see the problem.  All other parameters are offset by one, and you have to use the sixth (well, actually seventh because of the height) parameter.  In fact, using less than six parameters isn't even documented in `lua_api.txt`, so we don't actually have to worry about it.

Reply:
With "6 parameter element" I was referring to the tabheader element that only needs 6 parameters and ergo has no height.
But as you have added the following lines to the documentation, this is now fixed.
```
* **Important note**: This syntax for tabheaders can only be used with the
  old coordinate system.
```